resource/order.go

```diff
@@ -177,23 +147,13 @@ func (oe OrderEndpoint) GetOrderItems(orderId int) (model.OrderItems, error) {
 	}
 
 	rawBody := response.GetBody()
-	rawOrderItems, _, _, err := jsonparser.Get(rawBody, "OrderItems")
-
-	if err != nil {
-		return model.OrderItems{}, err
-	}
-
-	orderItems := model.OrderItems{}
-	if len(rawOrderItems) == 0 {
-		return orderItems, nil
-	}
-
-	err = json.Unmarshal(rawOrderItems, &orderItems)
+	var orderItems model.OrderItems
+	err = json.Unmarshal(rawBody, &orderItems)
```

Comment:
like the other comments please check err at first and if is nil return empty model.OrderItems{}